packages/clerk-js/src/v4/Components.tsx

```diff
@@ -63,24 +56,18 @@ function assertDOMElement(element: HTMLElement): asserts element {
 }
 
 export default class Components extends React.Component<ComponentsProps, ComponentsState> {
-  state: ComponentsState = {
-    signInModal: null,
-    signUpModal: null,
-    nodes: new Map(),
-  };
-
   static render(clerk: Clerk, environment: EnvironmentResource, options: ClerkOptions): Components {
     /**  Merge theme retrieved from the network with user supplied theme options. */
-    injectTheme(
-      environment.displayConfig.theme,
-      deepCamelToSnake(options.theme || {}) as DeepPartial<DisplayThemeJSON>,
-    );
+    // TODO: Remove all helpers
```

Comment:
@nikosdouvlis I'd suggest we do the network/prop merging in Clerk.ts to keep the Components renderer clean from external dependencies.

Reply:
This is part of a future PR of course.